mina.core/core/pom.xml

```diff
@@ -33,5 +33,14 @@
             <artifactId>easymockclassextension</artifactId>
         </dependency>
     </dependencies>
+	
+    <build>
```

Comment:
If this is truly in every pom (like it looks to me), lets just have it in the top level pom.  If it is not and only in the vast majority, lets have it in the top level pom and then use a [property](http://eclemma.org/jacoco/trunk/doc/prepare-agent-mojo.html#skip) to deactivate it in the poms it is not used in.  I think that would be cleaner.  Do you agree